downstream/assemblies/hub/assembly-managing-cert-valid-content.adoc

```diff
@@ -6,34 +6,42 @@ ifdef::context[:parent-context: {context}]
 :context: managing-cert-validated-content
 
 [role="_abstract"]
-{CertifiedName} are included in your subscription to {PlatformName}. Red Hat Ansible content includes two types of content: {CertifiedName} and {Valid}.
-Using {HubNameMain}, you can access and curate a unique set of collections from all forms of Ansible content.
+{CertifiedName} are included in your subscription to {PlatformName}. Using {HubNameMain}, you can access and curate a unique set of collections from all forms of Ansible content.
 
 Red Hat Ansible content contains two types of content:
 
 * {CertifiedName}
 * {Valid} collections
 
-Ansible validated collections are available in your {PrivateHubName} through the Platform Installer.
+You can use both {CertifiedName} or {Valid} collections to build your automation library. For more information on the differences between {CertifiedName} and {Valid} collections, see the Knowledgebase article link:https://access.redhat.com/support/articles/ansible-automation-platform-certified-content[{CertifiedName} and {Valid}], or xref:assembly-validated-content[{Valid}] in this guide.
+
+// hherbly--removed, see aap-20548  
+////
+Ansible validated collections are available in your {PrivateHubName} through the platform installer.
 When you download {PlatformName} with the bundled installer, validated content is pre-populated into the {PrivateHubName} by default, but only if you enable the {PrivateHubName} as part of the inventory.
```

Comment:
Is this enabled via an inventory file setting? If so, suggest changing to explicitly say so.... 

... but only when you enable {PrivateHubName} in the inventory file.

Reply:
hmm..this part was supposed to be commented out. I don't think this is true anymore, because validated content no longer has its own tarball on the download page.